tox.ini

```
@@ -12,17 +12,19 @@ package = skip
 ignore_errors = True
 deps =
     codespell~=2.0
-    ruff
+    ruff~=0.6.2
 commands =
+    ruff -V
```

I usually go with the longer form CLI arguments if they're being invoked programatically.
```suggestion
    ruff --version
```

---

Oh yes this is better!